config/envs/default.py

```diff
@@ -35,15 +35,22 @@ class DefaultConfig:
     # APIs Config
     TEST_APPLICATION_STORE_API_HOST = "http://application_store"
     TEST_FUND_STORE_API_HOST = "http://fund_store"
+    TEST_ACCOUNT_STORE_API_HOST = "http://account_store"
```

Comment:
Could we pull this in from common config? https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/config/commonconfig.py